### ISSUE TITLE
Remove `bottle :unneeded`

### ISSUE THREAD

**Issue by wendorf** [FairwindsOps/homebrew-tap#6]
Homebrew now adds this warning:

> Warning: Calling bottle :unneeded is deprecated! There is no replacement.

goreleaser removed `:unneeded` in https://github.com/goreleaser/goreleaser/pull/2591, but there hasn't been a release with that change yet.

**Comment by jnodorp-jaconi** [comment]
goreleaser/goreleaser#2591 has been released with [v0.183.0](https://github.com/goreleaser/goreleaser/releases/tag/v0.183.0).

**Comment by lucasreed** [comment]
Yes, we are working on this in our various projects, thanks for the heads up!
